app/models/journey.rb

```diff
@@ -59,4 +74,16 @@ class Journey < ApplicationRecord
            :cancelled?,
            :rejected?,
            to: :state_machine
+
+  def for_feed
+    attributes.slice(*FEED_ATTRIBUTES)
+      .merge(from_location.for_feed(prefix: :from))
+      .merge(to_location.for_feed(prefix: :to))
+      .merge(supplier.for_feed)
+      .merge(vehicle_registration: vehicle_registration)
+  end
+
+  def vehicle_registration
+    vehicle['registration']
```

Comment:
Vehicle can be nil so we'll need to handle that :)

Reply:
I'll handle it this way:
```
  def vehicle_registration
    vehicle['registration'] if vehicle
  end
```

(another way could be not adding the vehicle_registration key)